templates/avi/3/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+avi:
+  image: avinetworks/avi-rancher-controller:latest
```

Comment:
Don't use generic tags in docker images, not reproducible. Could you please use a concrete tag for docker image??
```
-
  image: avinetworks/avi-rancher-controller:latest
+  image: avinetworks/avi-rancher-controller:<VERSION_TAG>
```